command.py

```diff
@@ -173,6 +184,60 @@ def motd(motd_content: Optional[Dict[str, List[str]]], user: mantatail.UserConne
     user.send_string_to_client(motd_start_and_end["end_msg"])
 
 
+def process_channel_modes(
+    state: mantatail.ServerState, user: mantatail.UserConnection, args: List[str]
+) -> None:
+    supported_channel_modes = ["o"]
+    if args[0].lower() not in state.channels.keys():
+
+        error_no_such_channel(user, args[0])
+    elif len(args) == 1:
+
+        message = f'{irc_responses.RPL_CHANNELMODEIS} {args[0]} {" ".join(state.channels[args[0].lower()].modes)}'
+        user.send_string_to_client(message)
+    else:
+        assert len(args) == 3
+        target_chan = args[0]
+        mode_command = list(args[1])
+        target_user = args[2]
```

Comment:
Nit: there is a handy way to do all this.

```suggestion
        target_chan, mode_command, target_user = args
```

This makes `mode_command` a string, but the code below looks like it would work even with a string.